c2corg_ui/templates/helpers/view.html

```diff
@@ -120,20 +120,17 @@ <h3 class="text-center"><span translate>Merge documents</span> (<span x-translat
   </div>
 </%def>
 
-<%def name="show_delete_document_dialog()">\
+<%def name="show_delete_document_dialog(module)">\
+  <%
+      # get the module without the 's' at the end ('routes' -> 'route')
+      # to reuse existing translation strings
+      document_type = module[:-1]
```

Comment:
Perhaps. I have copied-paster what @tsauerwein has done for document merging :package:
The plural form is used (at least in the document merging) for instance to generate URL https://github.com/c2corg/v6_ui/blob/4ae76094a25c5b92e8d30a7950c8ea0e2854c7c1/c2corg_ui/static/partials/mergedocuments.html#L17

I have preferred to use a consistent approach for the deletion.